lib/wordpress/gravityForms/mutationSubmit.ts

```diff
@@ -0,0 +1,23 @@
+const mutationSubmitForm = `
```

Comment:
@Danishkhurshid Let's change this filename & const to mutationInsertFormEntry

When it makes sense I'd like to follow a similar naming style & structure that the WebDevStudios guys used since a majority of the interest for this project have come from that one, looking for a WP headless solution. It should help devs used to their project make a smooth transition... and their stuff is pretty clean.